src/BuildASub.java

```diff
@@ -60,4 +69,60 @@ public static String chooseMeat() throws Exception {
             throw new Exception("Meat not set!");
         }
     }
+
+    public static Boolean addSalad (){
+        Boolean subSalad = false;
```

Comment:
This is more of an OOP comment, but the word `sub` isn't needed here. The `addSalad` method can't see the other variables in the main method. This is one advantage of OOP programming